tests/unit/utils/get_mpt_version/test_get_mpt_version.yml

```diff
@@ -0,0 +1,5 @@
+- name: test_get_mpt_version
+  tags:
+    - test_get_mpt_version
+  exit_code: 0
+  command: python -m mgnify_pipelines_toolkit.utils.get_mpt_version
```

Comment:
Can you use a regex to validate the output?

Reply:
Good idea, would letting it match any combination of `*.*.*` where `*` has to be a digit be good enough? Cause otherwise there will always be some level of hardcoding

Reply:
Yeah, I think just trying to asserts any numbers is just fine